assets/js/modules/adsense/components/common/AdSenseConnectCTA/index.test.js

```diff
@@ -52,57 +47,60 @@ describe( 'AdSenseConnectCTA', () => {
 	describe( 'after click', () => {
 		let container;
 		beforeEach( () => {
-			container = render( <AdSenseConnectCTA />, { registry } ).container;
+			container = render(
+				<div>
+					<div id="adminmenu">
+						<a href="?page=googlesitekit-settings">Settings</a>
```

Comment:
This is minor, but this link stood out to me as odd when browsing the code. Is there any reason not to make it a full URL? It'd be more accurate anyway to what we'd encounter in production, and looks less weird at a glance 😅 

```suggestion
						<a href="http://test.test/wp-admin/admin.php?page=googlesitekit-settings">Settings</a>
```

Reply:
No reason not to at all, I have amended as per your suggestion!